hedera-node/hedera-evm-api/src/main/java/module-info.java

```diff
@@ -8,6 +8,8 @@
     requires tuweni.bytes;
     requires javax.inject;
     requires com.swirlds.common;
+    requires tuweni.units;
+    requires org.mockito.junit.jupiter;
```

Comment:
Why do we need to depend on mokito in a production module?  Shouldn't this be limited to test code?  We shouldn't ship test code in production.

Reply:
`requires org.mockito.junit.jupiter;` - REMOVED